public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts

```diff
@@ -111,18 +112,28 @@ export class ConceptChartsComponent implements OnChanges, OnDestroy {
     this.conceptName = this.concept.conceptName;
     this.subscriptions.push(this.api.getConceptAnalysisResults([conceptIdStr],
       this.concept.domainId).subscribe(
-      results => {
-        this.results = results.items;
-        this.analyses = results.items[0];
-        this.selectedSubGraph = 'Count';
-        this.selectedMeasurementType = 'No unit (Text)';
-        this.toDisplayGenderAnalysis = this.analyses.genderAnalysis;
-        this.toDisplayAgeAnalysis = this.analyses.ageAnalysis;
-        this.organizeGenders(this.analyses.genderAnalysis);
-        this.fetchMeasurementGenderResults();
-        // Set this var to make template simpler.
-        // We can just loop through the results and show bins
-        this.loadingStack.pop();
+      {
+        next: results => {
+          this.results = results.items;
+          this.analyses = results.items[0];
+          this.selectedSubGraph = 'Count';
+          this.selectedMeasurementType = 'No unit (Text)';
+          this.toDisplayGenderAnalysis = this.analyses.genderAnalysis;
+          this.toDisplayAgeAnalysis = this.analyses.ageAnalysis;
+          this.organizeGenders(this.analyses.genderAnalysis);
+          this.fetchMeasurementGenderResults();
+          // Set this var to make template simpler.
+          // We can just loop through the results and show bins
+          this.loadingStack.pop();
+          this.displayGraphErrorMessage = false;
+        },
+        error: err => {
+          const errorBody = JSON.parse(err._body);
+          if (errorBody.statusCode === 500) {
```

Comment:
Do we need to show an error only for 500s or for all errors? I see 400s and 503s on the workbench fairly often

Reply:
I just checked errors in stackdriver and they are always 500 very rare 503. We do not have any 400 errors for now.

Reply:
I agree the 500 should catch most cases, but I guess I'm wondering if we need to filter the errors at all. The error message is pretty generic, so why not remove the if statement and show the message no matter what type of error is returned? Otherwise, in the unlikely event you get a 400 or other error, the UI will break and the user will not get notified that an error occurred.

Reply:
Agree removed the check. Thanks.